backend/app/main.py

```
@@ -0,0 +1,115 @@
+    session.commit()
+    session.refresh(entry)
+    return entry
+
+
+@app.post('/audio/transcribe', response_model=EntryOut)
+async def transcribe_audio(
+    audio: UploadFile,
+    project: str | None = None,
+    task: str | None = None,
+    context: str | None = None,
+    session: Session = Depends(get_session),
```

Ruff (0.15.9)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)